src/pages/Main/ImgCom.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+class ImgCom extends Component {
+  render() {
+    const { src, alt } = this.props;
+    return (
+      <>
+        <img src={src} alt={alt}></img>
+      </>
```

Comment:
- 이미 img 태그 하나만 있기 때문에 불필요한 fragment 입니다! 
- 태그 안에 들어가는 컨텐츠가 없다면 셀프클로징 해주세요.

```suggestion
        <img src={src} alt={alt} />
```